src/Compilers/Core/Portable/PEWriter/Members.cs

```diff
@@ -992,15 +992,36 @@ public static bool ShouldInclude(this ITypeDefinitionMember member, EmitContext
                 return true;
             }
 
+            bool acceptBasedOnVisibility = true;
+
             switch (member.Visibility)
             {
                 case TypeMemberVisibility.Private:
-                    return context.IncludePrivateMembers;
+                    acceptBasedOnVisibility = context.IncludePrivateMembers;
```

Comment:
>context.IncludePrivateMembers [](start = 46, length = 29)

When would this ever be anything but false? We already did a shortcut out at the beginning of this method if this was true. It seems like the only thing we need to check is IVT for Assembly and FamilyAndAssembly members.

Reply:
> When would this ever be anything but false? We already did a shortcut out at the beginning of this method if this was true. It seems like the only thing we need to check is IVT for Assembly and FamilyAndAssembly members.

It looks like you are correct, but I prefer not changing the logic here.